Package.swift

```diff
@@ -21,6 +21,7 @@ let package = Package(
         .library(name: "BloomFilterWrapper", targets: ["BloomFilterWrapper"]),
         .library(name: "UserScript", targets: ["UserScript"]),
         .library(name: "Crashes", targets: ["Crashes"]),
+        .library(name: "cxxCrashHandler", targets: ["cxxCrashHandler"]),
```

Comment:
Xcode complaints about naming on my machine:

<img width="525" alt="cxx" src="https://github.com/duckduckgo/BrowserServicesKit/assets/2865434/71c69d6a-1ede-44ea-97c2-388d1caf9f68">

Reply:
I double renamed it to a capitalized form, could you re-check